static/js/publisher/market/market.js

```diff
@@ -105,12 +105,8 @@ function initSnapScreenshotsEdit(screenshotsToolbarElId, screenshotsWrapperElId,
       return '';
     }
     return `<p>
-      ${newCount
-          ? newCount + ' image' + (newCount > 1 ? 's' : '') + ' to upload. '
-          : ''}
-      ${deleteCount
-          ? deleteCount + ' image' + (deleteCount > 1 ? 's' : '') + ' to delete.'
-          : ''}
+      ${newCount ? newCount + ' image' + (newCount > 1 ? 's' : '') + ' to upload. ' : ''}
```

Comment:
Turns out we had some lint errors that didn't show up earlier (not sure why?) - fixing them now.